.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+language: go
+go_import_path: go.uber.org/goleak
+
+go:
+  # TODO(prashant): Enable once we have fixed Linux issues with old versions.
+  #- 1.6
+  #- 1.7
+  #- 1.8
+  - 1.9
+
+matrix:
+  include:
+  - go: 1.9
+    env: NO_TEST=yes COVERAGE=yes LINT=yes
+
+cache:
+  directories:
+    - vendor
+
+install:
+  - make install
+
+script:
+  - test -n "$NO_TEST" || make test
+  - test -z "$COVERAGE" || scripts/coverage.sh
+  - test -n "$LINT" || make install_lint lint
```

Comment:
I think you meant to use `-z "$LINT"` here.

Reply:
👍 thanks